Documentation/gettingstarted/bgp.rst

```diff
@@ -60,7 +60,9 @@ Here are the required Helm values:
 
 .. parsed-literal::
 
-   helm install cilium |CHART_RELEASE| --set bgp.enabled=true --set bgp.announce.loadbalancerIP=true
+   helm install cilium |CHART_RELEASE| --set bgp.enabled=true --set bgp.announce.loadbalancerIP=true --set bgp.announce.podCIDR=true
```

Comment:
Note: You may get a conflict from https://github.com/cilium/cilium/pull/17496 on this line.

Reply:
you called it @qmonnet  ;) I'll rebase.